src/Components/BehaviorsModal/Internal/BehaviorSection/BehaviorSection.tsx

```diff
@@ -77,30 +77,33 @@ const BehaviorSection: React.FC<IBehaviorsSectionProps> = ({ behavior }) => {
 const AlertBlock: React.FC<{ alertVisual: IExpressionRangeVisual }> = ({
     alertVisual
 }) => {
-    const styles = getStyles();
-
-    const {
-        visual: { color, iconName, labelExpression }
-    } = alertVisual.valueRanges[0];
+    if (alertVisual.valueRanges[0]) {
```

Comment:
nit: shorter way to do this is to check the inverse, then you don't have to wrap the rest of the render block. 
```jsx
if(!myCheck){
  return null
}
```